sdks/python/apache_beam/examples/datastore_wordcount.py

```diff
@@ -61,53 +65,134 @@ def process(self, context):
     return words
 
 
+class CreateEntityFn(object):
+  """Create a Cloud Datastore entity from the given string."""
+  def __init__(self, namespace, kind, ancestor):
+    self._namespace = namespace
+    self._kind = kind
+    self._ancestor = ancestor
+
+  def make_entity(self, content):
+    entity = entity_pb2.Entity()
+    if self._namespace is not None:
```

Comment:
namespace can be an empty string`''` to represent the `default` namespace, hence we need to explicitly check for `None`